tables/de-de-comp8.ctb

```diff
@@ -276,18 +277,191 @@ sign \x00fc 12568
 sign \x00fd 24568
 sign \x00fe 12348
 sign \x00ff 134568
+sign \x25e6 67		# white bullet
+sign \x25fe 3678        # black square
 sign \x2022 35		# •
 sign \x2014 36		# —
 sign \x2013 36		# –
 sign \x201e 1268	# „
 sign \x201c 138		# “
 sign \x201a 3678	# ‚
 sign \x2018 2367	# ‘
-noback sign \X25CF 35	# 9679 black circle
```

Comment:
Why was this line removed?

Reply:
Wasn't this added by Victor Montalvão? Maybe we should have a test for this?

Reply:
Ah, yes you're right. I'm getting a sneaking suspicion that the file that was attached to #677 is just a revert of cc72133b6dcf2a6d72a5657d7a8793f1e343db49. But then again if that is Aliminator83@gmail.com that is doing this then I guess we have his implicit consent to re-license which means we can revert.

Reply:
If you are gonna put non-standard mappings in it should be made very clear with comments within the table and in the table header. I don't think this is the case yet.

Reply:
I added some comments about the standards compliance

Reply:
Thanks.